### ISSUE TITLE
Chat is slow checking if user is muted

### ISSUE THREAD

**Issue by Turpster** [TheMite-Server/TMCore#5]

**Comment by Turpster** [comment]
This issue is derived from #1

**Comment by Turpster** [comment]
Could you show me a few examples of this issue? I am having a hard time understanding what is slow.

**Comment by cjcameron92** [comment]
yeah for sure, chat is basically instead of sending a message instantly it is delayed around 2 seconds.

**Comment by Turpster** [comment]
Alright, this needs more research in figuring out what the problem is...
